xarray/backends/rasterio_.py

```diff
@@ -141,7 +141,19 @@ def open_rasterio(filename, chunks=None, cache=None, lock=None):
         # CRS is a dict-like object specific to rasterio
         # We convert it back to a PROJ4 string using rasterio itself
         attrs['crs'] = riods.crs.to_string()
-    # Maybe we'd like to parse other attributes here (for later)
+    if hasattr(riods, 'res'):
+        # (width, height) tuple of pixels in units of CRS
+        attrs['res'] = riods.res
+    if hasattr(riods, 'tiled') or \
```

Comment:
can you use parentheses here to group conditions instead of the explicit `\`? This is preferred, per PEP8.